app.rb

```diff
@@ -63,7 +63,10 @@
           pull_request.add_comment(message)
         end
       elsif !issue.rejected?
-        issue.set_triaged(false) if issue.backlog? || issue.recycle_bin? || issue.version.nil?
+        if issue.backlog? || issue.recycle_bin? || issue.version.nil?
+          issue.set_triaged(false)
+          issue.set_target_version(nil)
```

Comment:
I know its a bit redundant to unset the version on `issue.version.nil?` but it kept the logic much simpler to just do both these things in a block and it doesn't seem like a big cost. I'm open to other solutions though :wink:

Reply:
It's good to be explicit about the desired state and doesn't hit a remote API anyway so it's very cheap.